ci/Dockerfile

```diff
@@ -1,17 +1,23 @@
-# This file was inspired by envoy Dockerfile:
-# https://github.com/envoyproxy/envoy/blob/445a67344ffda0c8828c8e438e463fcaa7878434/ci/Dockerfile-envoy-alpine
-
-FROM frolvlad/alpine-glibc:alpine-3.17_glibc-2.34
+# This file was inspired by mesh Dockerfile:
+# https://github.com/solo-io/gloo-mesh-enterprise/blob/c40b21c3ef260de1acc92ba25bd1794b17074c8c/docker/Dockerfile.ubuntu#L4
+# as it previously was based off of envoys old alpine setup. 
+FROM ubuntu:focal-20220826
```

Comment:
Thanks for including the link to the mesh Dockerfile above. Based on the Ubuntu Docker hub page, it looks like `focal-20230801` is the latest 20.04 image release -- do you think it makes sense to bump to that, or is it more important to use the same version as platform? https://hub.docker.com/_/ubuntu